index.html

```diff
@@ -11,13 +11,13 @@
 <body class="body">
 
   <div class="navbar">
-    <div class="my-logo">My Logo</div>
+    <a class="my-logo">My Logo</a>
     <a class="menu-icon"></a>
   </div>
 
   <div class="main">
```

Comment:
- [ ] Please make this a `section` tag and not a `div` tag. Currently you are working on the introduction section and it would be semantic if youenslosed it in a `section` tag instead 👍